src/modules/31-filestore/components/FileStoreComponent/FileStoreComponent.tsx

```diff
@@ -78,7 +80,8 @@ const useFileStoreModal = ({ applySelected, fileUsage }: UseFileStoreModalProps)
   const commonProps = {
     isModalView: true,
     onNodeChange: handleSelectFile,
-    fileUsage
+    fileUsage,
+    handleSetIsUnsaved: (status: boolean) => setIsUnsaved(status)
```

Comment:
`setIsUnsaved` can be passed directly as this function just takes the `status` and calls `setIsUnsaved` 
```suggestion
    setIsUnsaved
```